src/Effects.js

```diff
@@ -430,6 +430,113 @@ module.exports = class Effects {
       }
     };
 
+    this.kaleidoscope = {
+      init: function () {
```

Comment:
Do we have any kind of `unload` for backgrounds?  Two 200x200 buffers (80kb) is not going to bust our memory budget, just wondering if in general we're able to clean up when the background is no longer in use.

Reply:
Nothing yet, but I think we should look into this as part of the background crashes on real devices.